tokio/src/stream/mod.rs

```diff
@@ -56,6 +56,12 @@ use take::Take;
 mod take_while;
 use take_while::TakeWhile;
 
+cfg_time! {
+    mod timeout;
+    pub use timeout::Timeout;
```

Comment:
As of _now_ we aren't re-exporting combinator types (this may change).

```suggestion
    use timeout::Timeout;
```

Reply:
Done